src/modules/user/domain/models/user.model.ts

```diff
@@ -1,7 +1,22 @@
 import { AggregateRoot } from '@nestjs/cqrs';
 
+import { UserCreatedEvent } from '@user/domain/events/events-success-domain/user-created.event';
 import { IUserSchema } from '@user/domain/schemas/user.schema';
 
 export class UserModel extends AggregateRoot {
   private readonly _entityRoot: IUserSchema;
+
+  get id(): number {
+    return this._entityRoot.id._value;
+  }
+  get uuid(): string {
+    return this._entityRoot.uuid._value;
+  }
+  get status(): string {
+    return this._entityRoot.status;
```

Comment:
Aqui tambien deberia ser igual que los otros, no deja de ser un value object

```suggestion
    return this._entityRoot.status._value;
```

Reply:
@2314-RAustin Listo, corregido.